### PR TITLE
Add TENKI keyboard

### PR DESCRIPTION
**Pull request by eriqadams**
Add TENKI keyboard, default keymap and via keymap

## Description

Add new TENKI keyboard: default keymap and via keymap
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tenki/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  
+  [2] = LAYOUT_ortho_20(
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS
+  ),
+  
+  [3] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/via/keymap.c:28]:
```suggestion
  [3] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  [1] = LAYOUT_ortho_20(
+    KC_TRNS, 	RGB_HUI,    RGB_HUD,    	RGB_VAD,\
+    KC_TRNS, 	KC_VOLU,    RGB_TOG,    	RGB_VAI,\
+    KC_MPRV, 	KC_MUTE,    KC_MNXT,    	KC_TRNS,\
+    KC_TRNS, 	KC_VOLD,    KC_TRNS,    	KC_TRNS,\
+    RESET, 		KC_TRNS,    KC_TRNS,    	RGB_MOD
+  ),
+  
+  [2] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/via/keymap.c:20]:
```suggestion
  [2] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_20(
+    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,\
+    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,\
+    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,\
+    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,\
+    KC_P0 , 	KC_PSCR, 	LT(1,KC_PDOT) , KC_PENT
+  ),
+
+  [1] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/via/keymap.c:12]:
```suggestion
  [1] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,38 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/via/keymap.c:4]:
```suggestion
  [0] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_20(
+    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,\
+    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,\
+    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,\
+    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,\
+    KC_P0 , 	KC_PSCR , 	LT(1,KC_PDOT) , KC_PENT
+  ),
+
+  [1] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/default/keymap.c:12]:
```suggestion
  [1] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_20(
```
Comment (zvecr) [inline keyboards/tenki/keymaps/default/keymap.c:4]:
```suggestion
  [0] = LAYOUT_ortho_5x4(
```

File: keyboards/tenki/rules.mk [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+EXTRAKEY_ENABLE = yes	  # Audio control and System control
+CONSOLE_ENABLE = no			# Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes				# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+
+LAYOUTS = ortho_20
```
Comment (zvecr) [inline keyboards/tenki/rules.mk:27]:
```suggestion
LAYOUTS = ortho_5x4
```

File: keyboards/tenki/tenki.h [hunk 8]
```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_20( \
```
Comment (zvecr) [inline keyboards/tenki/tenki.h:5]:
```suggestion
#define LAYOUT_ortho_5x4( \
```

File: keyboards/tenki/info.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "TENKI", 
+    "url": "https://tokopedia.com/pixlup", 
+    "maintainer": "eriqadams", 
+    "width": 4, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
+            "layout_ortho_20": [{"label":"Num Lock", "x":0, "y":0}, {"label":"/", "x":1, "y":0}, {"label":"*", "x":2, "y":0}, {"label":"-", "x":3, "y":0}, {"label":"7", "x":0, "y":1}, {"label":"8", "x":1, "y":1}, {"label":"9", "x":2, "y":1}, {"label":"+", "x":3, "y":1}, {"label":"4", "x":0, "y":2}, {"label":"5", "x":1, "y":2}, {"label":"6", "x":2, "y":2}, {"label":"Bspc", "x":3, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"=", "x":3, "y":3}, {"label":"0", "x":0, "y":4}, {"label":"PrtScr", "x":1, "y":4}, {"label":".", "x":2, "y":4}, {"label":"Enter", "x":3, "y":4}]
+        }
```
Comment (zvecr) [inline keyboards/tenki/info.json:10]: Invalid format of this file
```suggestion
        "LAYOUT_ortho_5x4": [{"label":"Num Lock", "x":0, "y":0}, {"label":"/", "x":1, "y":0}, {"label":"*", "x":2, "y":0}, {"label":"-", "x":3, "y":0}, {"label":"7", "x":0, "y":1}, {"label":"8", "x":1, "y":1}, {"label":"9", "x":2, "y":1}, {"label":"+", "x":3, "y":1}, {"label":"4", "x":0, "y":2}, {"label":"5", "x":1, "y":2}, {"label":"6", "x":2, "y":2}, {"label":"Bspc", "x":3, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"=", "x":3, "y":3}, {"label":"0", "x":0, "y":4}, {"label":"PrtScr", "x":1, "y":4}, {"label":".", "x":2, "y":4}, {"label":"Enter", "x":3, "y":4}]
```

File: keyboards/tenki/keymaps/via/keymap.c (hunk 1 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/via/keymap.c]: Done

File: keyboards/tenki/keymaps/via/keymap.c (hunk 2 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/via/keymap.c]: Done

File: keyboards/tenki/keymaps/via/keymap.c (hunk 3 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/via/keymap.c]: Done

File: keyboards/tenki/keymaps/via/keymap.c (hunk 4 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/via/keymap.c]: Done

File: keyboards/tenki/keymaps/default/keymap.c (hunk 5 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/default/keymap.c]: Done

File: keyboards/tenki/keymaps/default/keymap.c (hunk 6 above)
Comment (eriqadams) [reply keyboards/tenki/keymaps/default/keymap.c]: Done

File: keyboards/tenki/rules.mk (hunk 7 above)
Comment (eriqadams) [reply keyboards/tenki/rules.mk]: Done

File: keyboards/tenki/tenki.h (hunk 8 above)
Comment (eriqadams) [reply keyboards/tenki/tenki.h]: Done

File: keyboards/tenki/info.json (hunk 9 above)
Comment (eriqadams) [reply keyboards/tenki/info.json]: Done

File: keyboards/tenki/tenki.h [hunk 10]
```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_5x4( \
+	K00,   K01,   K02,   K03, \
```
Comment (noroadsleft) [inline keyboards/tenki/tenki.h:6]:
```suggestion
    K00,   K01,   K02,   K03, \
```

Comment (eriqadams) [reply keyboards/tenki/tenki.h]: ok done. thanks :)

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**